server/images.go

```diff
@@ -862,7 +862,7 @@ func downloadBlob(mp ModelPath, digest string, username, password string, fn fun
 		})
 
 		if completed >= total {
-			if err := os.Rename(fp+"-partial", fp); err != nil {
+			if err := os.Symlink(fp+"-partial", fp); err != nil {
```

Comment:
wouldn't this leave the "-partial" file around?

Reply:
that's ok. the completed file is checked first. if it exists, no further attempt is made to download the file

Reply:
what was the reason for using symlink over closing the file then renaming? My instinct is to prefer the rename, so Im using the actual files in general rather than symlinks (one less layer of indirection)

Reply:
IMO it's cleaner but it turns out Windows require privilege so rolling back this change to `Close` and `Rename`